documentation/content/en/books/fdp-primer/writing-style/_index.adoc

```diff
@@ -321,7 +352,7 @@ This rule proposes better wording.
 
 . FreeBSD.Hyphens: Often adverbs ending with 'ly' are added with a hyphen which is wrong.
 
-. FreeBSD.Spacing: Often double spaces are hard to catch with the naked eye and this is addressed here.
+. FreeBSD.Spacing: Between a pair of sentences, it may be incorrect to have two spaces (see link:{sentence-spacing}[spacing between sentences]). 
```

Comment:
<https://github.com/freebsd/freebsd-doc/tree/main/.vale/styles/FreeBSD>

- I can not see a related `.yml` file
- am I looking in the wrong place?